### ISSUE TITLE
The button to expand and collabs the replace portion of the find all dialog box is unlabeled

### ISSUE THREAD

**Issue by venkateshpotluri** [microsoft/vscode#221852]
Steps to reproduce:
1. in any document, press controll+shift+f to open the find all functionality.
1. press tab (4 times) until you hear "button" or "button expanded".

Issues:
1. JAWS does not announce that the button is an expandable one, and that it is in the collapsed state. This appears to be a regression in the insiders build when compared with the latest stable release, where the button is announced as collapsed or expanded.
2. JAWS does not have any label for the button. It is just announced as "button".

**Comment by andreamah** [comment]
@meganrogge are you aware of any regressions that could have caused this in the tree view? I don't think that I changed anything in search for this to happen..

**Comment by meganrogge** [comment]
I'm not aware of any changes to this. Furthermore, in testing with VoiceOver, this is working as expected. Does this repro with NVDA on Windows? If not, it's possible something changed with JAWS.

**Comment by meganrogge** [comment]
This doesn't repro with NVDA, trying JAWS next

**Comment by meganrogge** [comment]
doesn't repro for me using JAWS either, @venkateshpotluri has any setting changed for you?

**Comment by meganrogge** [comment]
The expand/collapsed state is indicated to me for all screen readers. 

I can repro the button not having an aria label, which we should fix.
